egs/puebla_nahuatl/st1/local/score_bleu.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
```

Comment:
This file is not necessary. This is a scoring script specific to fisher-callhome.